zooniverse/front-end-monorepo
# app-content-pages: Built a shared sticky side bar component

~~⚠️ This contains #5488 and #5551 please review those first~~

## Package
app-content-pages

## Linked Issue and/or Talk Post
Toward: https://github.com/zooniverse/front-end-monorepo/issues/5476
Project board: https://github.com/orgs/zooniverse/projects/40
Figma design: https://www.figma.com/file/zN9InDg8CTk2vnWp406moK/About-Pages

## Describe your changes

- The new designs for the About Zooniverse pages include a "scroll to" UX rather than filtered data. I refactored the side bar links to bring a user to the linked section (publications category or team location) rather than filter the data and re-render. The side bar is now sticky, too.
- I extracted those side bar links to a Sidebar component because the same design is used in the root About page (to be built). I added a story and unit tests for that Sidebar.
- I removed the TwoColumnLayout component in favor of styling the Teams and Publications pages directly in their files.
- I removed the TeamsContainer and PublicationsContainer files because their filtering logic is no longer needed

## How to Review

- Run app-content-pages storybook and view the new Sidebar story
- Run app-content-pages locally and check out https://local.zooniverse.org:3000/about/publications and https://local.zooniverse.org:3000/about/teams. Click the side bar to scroll to a section
- I would like to explore the Intersection Observer API for detecting when a user has reached a certain section while scrolling down, and adjust the active section link in the sidebar (and the url?)
- There may be a few design adjustments - I'll tag Sean here with a couple of a UI questions

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## New Feature
  - Example of SlideTutorial [component](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js) with docgen comments, and its [story](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/docs/other-slidetutorial--default)

## Refactoring
